packages/vite/src/index.js

```diff
@@ -1,37 +1,37 @@
 /** @returns {import('vite').Plugin} */
 export default function prefreshPlugin() {
-    return {
-        transforms: [
-            {
-                as: 'js',
-                test(path, query) {
-                    if (!/\.[tj]sx$/.test(path)) return false;
-                    // @ts-ignore
-                    this.path = path;
-                    return true;
-                },
-                // todo: use isImport to ignore `/@modules` requests?
-                transform(code, isImport) {
-                    // The module imports itself to get its current exports, avoiding the need for eval().
+	return {
+		transforms: [
+			{
+				as: 'js',
+				test(path, query) {
+					if (!/\.[tj]sx$/.test(path)) return false;
+					// @ts-ignore
+					this.path = path;
+					return true;
+				},
+				// todo: use isImport to ignore `/@modules` requests?
+				transform(code, isImport) {
+					// The module imports itself to get its current exports, avoiding the need for eval().
 
-                    // @ts-ignore
-                    const spec = JSON.stringify(this.path);
+					// @ts-ignore
+					const spec = JSON.stringify(this.path);
 
-                    // Note: prefresh *must* be injected prior to any VNodes being created!
-                    return `
-                        import 'prefresh/packages/core';
-                        import { hot } from 'vite/hmr';
-                        import * as __PSELF__ from ${spec};
-                        ${code}
-                        if (__DEV__) {
-                            let a = 0;
-                            hot.accept(m => {
-                                if (!a++) for (let i in m) self.__PREFRESH__.replaceComponent(__PSELF__[i], m[i]);
-                            });
-                        }
-                    `;
-                }
+					// Note: prefresh *must* be injected prior to any VNodes being created!
+					return `
+            import 'prefresh/packages/core';
```

Comment:
Can we use `@prefresh/core` since this is a dependency

Reply:
ack! yes. it's not published though :P

Reply:
It should be published